source/extensions/listener_managers/listener_manager/connection_handler_impl.cc

```diff
@@ -245,7 +245,7 @@ void ConnectionHandlerImpl::stopListeners(uint64_t listener_tag) {
   if (auto iter = listener_map_by_tag_.find(listener_tag); iter != listener_map_by_tag_.end()) {
     iter->second->invokeListenerMethod([](Network::ConnectionHandler::ActiveListener& listener) {
       if (listener.listener() != nullptr) {
-        listener.shutdownListener();
+        listener.shutdownListener(nullptr);
```

Comment:
nit: do we need the `nullptr` here since I *think* that's the default? (I'm also happy to not have default arguments and instead be explicit everywhere)

Reply:
Removed the default, and now it's {} because the parameter isn't a pointer.